tests/pages/services/ServiceActions-cy.js

```diff
@@ -391,7 +391,8 @@ describe("Service Actions", function() {
       cy.route({
         method: "PUT",
         url: /marathon\/v2\/apps\/\/cassandra-healthy/,
-        response: []
+        response: [],
+        delay: 100
```

Comment:
Any idea what transition makes these delays necessary?

Reply:
It's related to the modal. I added these as a troubleshooting step to try and diagnose the issue. Testing the button is disabled while the API call is made. and all these tests pass locally.

Reply:
Ah that makes sense. Thanks